auth_backends/tests/test_backends.py

```diff
@@ -310,3 +311,21 @@ def test_end_session_url(self):
         # Now, add the public url root to the settings.
         self.set_social_auth_setting('PUBLIC_URL_ROOT', self.public_url_root)
         self.assertEqual(self.backend.end_session_url(), self.public_url_root + logout_location)
+
+    def test_user_data(self):
+        user_data = self.backend.user_data(self.create_jws_access_token())
+        self.assertDictEqual(user_data, {
+            'name': 'Joe Smith',
+            'preferred_username': 'jsmith',
+            'email': 'jsmith@example.com',
+            'given_name': 'Joe',
+            'user_id': '1',
+            'family_name': 'Smith',
+            'administrator': False
+        })
+
+    def test_extra_data(self):
+        """
+        Ensure that `user_id` stays in EXTRA_DATA.
+        """
+        self.assertEqual(self.backend.EXTRA_DATA, [('user_id', 'user_id', True)])
```

Comment:
FYI: I didn't want to do an integration test, and I want to ensure this doesn't get lost, so I added a silly simple test.